[ 2024-04-19 05:28:11,417 ] 94 main_controller - INFO - Subprocess 0 started with trial number 0
[ 2024-04-19 05:28:11,417 ] 94 main_controller - ERROR - Account is a dictionary, convert to Account object: Username: Nickpown0411@gmail.com, Password: hoahong1234
[ 2024-04-19 05:28:13,422 ] 94 main_controller - INFO - Subprocess 1 started with trial number 0
[ 2024-04-19 05:28:13,422 ] 94 main_controller - INFO - Subprocess 1 started with trial number 0
[ 2024-04-19 05:28:13,422 ] 94 main_controller - ERROR - Account is a dictionary, convert to Account object: Username: marjyamada040493@gmail.com, Password: Marj0409
[ 2024-04-19 05:28:13,422 ] 94 main_controller - ERROR - Account is a dictionary, convert to Account object: Username: marjyamada040493@gmail.com, Password: Marj0409
[ 2024-04-19 05:28:13,445 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:13,445 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:14,738 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:14,738 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:15,444 ] 94 main_controller - INFO - Subprocess 2 started with trial number 0
[ 2024-04-19 05:28:15,444 ] 94 main_controller - INFO - Subprocess 2 started with trial number 0
[ 2024-04-19 05:28:15,444 ] 94 main_controller - INFO - Subprocess 2 started with trial number 0
[ 2024-04-19 05:28:15,445 ] 94 main_controller - ERROR - Account is a dictionary, convert to Account object: Username: Jamebrown0206@gmail.com, Password: hoahong1234
[ 2024-04-19 05:28:15,445 ] 94 main_controller - ERROR - Account is a dictionary, convert to Account object: Username: Jamebrown0206@gmail.com, Password: hoahong1234
[ 2024-04-19 05:28:15,445 ] 94 main_controller - ERROR - Account is a dictionary, convert to Account object: Username: Jamebrown0206@gmail.com, Password: hoahong1234
[ 2024-04-19 05:28:17,915 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:17,915 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:17,915 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:24,395 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:24,395 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:24,395 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:24,600 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Nickpown0411.png
[ 2024-04-19 05:28:24,600 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Nickpown0411.png
[ 2024-04-19 05:28:24,600 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Nickpown0411.png
[ 2024-04-19 05:28:24,601 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:24,601 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:24,601 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:25,694 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:25,694 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:25,694 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:25,914 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_marjyamada040493.png
[ 2024-04-19 05:28:25,914 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_marjyamada040493.png
[ 2024-04-19 05:28:25,914 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_marjyamada040493.png
[ 2024-04-19 05:28:25,914 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:25,914 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:25,914 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:28,118 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:28,118 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:28,118 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:28,357 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Jamebrown0206.png
[ 2024-04-19 05:28:28,357 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Jamebrown0206.png
[ 2024-04-19 05:28:28,357 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Jamebrown0206.png
[ 2024-04-19 05:28:28,357 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:28,357 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:28,357 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:30,218 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:30,218 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:30,218 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:30,227 ] 94 main_controller - INFO - Subprocess 0 started with trial number 1
[ 2024-04-19 05:28:30,227 ] 94 main_controller - INFO - Subprocess 0 started with trial number 1
[ 2024-04-19 05:28:30,227 ] 94 main_controller - INFO - Subprocess 0 started with trial number 1
[ 2024-04-19 05:28:30,227 ] 94 main_controller - INFO - Subprocess 0 started with trial number 1
[ 2024-04-19 05:28:31,431 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:31,431 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:31,431 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:31,431 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:34,094 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:34,094 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:34,094 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:34,094 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:34,100 ] 94 main_controller - INFO - Subprocess 1 started with trial number 1
[ 2024-04-19 05:28:34,100 ] 94 main_controller - INFO - Subprocess 1 started with trial number 1
[ 2024-04-19 05:28:34,100 ] 94 main_controller - INFO - Subprocess 1 started with trial number 1
[ 2024-04-19 05:28:34,100 ] 94 main_controller - INFO - Subprocess 1 started with trial number 1
[ 2024-04-19 05:28:34,100 ] 94 main_controller - INFO - Subprocess 1 started with trial number 1
[ 2024-04-19 05:28:35,727 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:35,727 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:35,727 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:35,727 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:35,727 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:36,695 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:36,695 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:36,695 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:36,695 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:36,695 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:36,699 ] 94 main_controller - INFO - Subprocess 2 started with trial number 1
[ 2024-04-19 05:28:36,699 ] 94 main_controller - INFO - Subprocess 2 started with trial number 1
[ 2024-04-19 05:28:36,699 ] 94 main_controller - INFO - Subprocess 2 started with trial number 1
[ 2024-04-19 05:28:36,699 ] 94 main_controller - INFO - Subprocess 2 started with trial number 1
[ 2024-04-19 05:28:36,699 ] 94 main_controller - INFO - Subprocess 2 started with trial number 1
[ 2024-04-19 05:28:36,699 ] 94 main_controller - INFO - Subprocess 2 started with trial number 1
[ 2024-04-19 05:28:39,044 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:39,044 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:39,044 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:39,044 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:39,044 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:39,044 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:42,154 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:42,154 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:42,154 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:42,154 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:42,154 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:42,154 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:42,449 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Nickpown0411.png
[ 2024-04-19 05:28:42,449 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Nickpown0411.png
[ 2024-04-19 05:28:42,449 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Nickpown0411.png
[ 2024-04-19 05:28:42,449 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Nickpown0411.png
[ 2024-04-19 05:28:42,449 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Nickpown0411.png
[ 2024-04-19 05:28:42,449 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Nickpown0411.png
[ 2024-04-19 05:28:42,451 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:42,451 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:42,451 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:42,451 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:42,451 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:42,451 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:45,606 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:45,606 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:45,606 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:45,606 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:45,606 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:45,606 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:45,812 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_marjyamada040493.png
[ 2024-04-19 05:28:45,812 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_marjyamada040493.png
[ 2024-04-19 05:28:45,812 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_marjyamada040493.png
[ 2024-04-19 05:28:45,812 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_marjyamada040493.png
[ 2024-04-19 05:28:45,812 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_marjyamada040493.png
[ 2024-04-19 05:28:45,812 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_marjyamada040493.png
[ 2024-04-19 05:28:45,813 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:45,813 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:45,813 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:45,813 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:45,813 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:45,813 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:49,049 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:49,049 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:49,049 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:49,049 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:49,049 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:49,049 ] 94 main_controller - INFO - Logged in successfully
[ 2024-04-19 05:28:49,250 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Jamebrown0206.png
[ 2024-04-19 05:28:49,250 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Jamebrown0206.png
[ 2024-04-19 05:28:49,250 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Jamebrown0206.png
[ 2024-04-19 05:28:49,250 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Jamebrown0206.png
[ 2024-04-19 05:28:49,250 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Jamebrown0206.png
[ 2024-04-19 05:28:49,250 ] 94 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_19\05_28_10\screenshot_Jamebrown0206.png
[ 2024-04-19 05:28:49,251 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:49,251 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:49,251 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:49,251 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:49,251 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:49,251 ] 94 main_controller - INFO - Finding forms
[ 2024-04-19 05:28:50,042 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:50,042 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:50,042 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:50,042 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:50,042 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:50,042 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:50,044 ] 94 main_controller - INFO - Subprocess 0 started with trial number 2
[ 2024-04-19 05:28:50,044 ] 94 main_controller - INFO - Subprocess 0 started with trial number 2
[ 2024-04-19 05:28:50,044 ] 94 main_controller - INFO - Subprocess 0 started with trial number 2
[ 2024-04-19 05:28:50,044 ] 94 main_controller - INFO - Subprocess 0 started with trial number 2
[ 2024-04-19 05:28:50,044 ] 94 main_controller - INFO - Subprocess 0 started with trial number 2
[ 2024-04-19 05:28:50,044 ] 94 main_controller - INFO - Subprocess 0 started with trial number 2
[ 2024-04-19 05:28:50,044 ] 94 main_controller - INFO - Subprocess 0 started with trial number 2
[ 2024-04-19 05:28:51,405 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:51,405 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:51,405 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:51,405 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:51,405 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:51,405 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:51,405 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:53,379 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:53,379 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:53,379 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:53,379 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:53,379 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:53,379 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:53,379 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:53,388 ] 94 main_controller - INFO - Subprocess 1 started with trial number 2
[ 2024-04-19 05:28:53,388 ] 94 main_controller - INFO - Subprocess 1 started with trial number 2
[ 2024-04-19 05:28:53,388 ] 94 main_controller - INFO - Subprocess 1 started with trial number 2
[ 2024-04-19 05:28:53,388 ] 94 main_controller - INFO - Subprocess 1 started with trial number 2
[ 2024-04-19 05:28:53,388 ] 94 main_controller - INFO - Subprocess 1 started with trial number 2
[ 2024-04-19 05:28:53,388 ] 94 main_controller - INFO - Subprocess 1 started with trial number 2
[ 2024-04-19 05:28:53,388 ] 94 main_controller - INFO - Subprocess 1 started with trial number 2
[ 2024-04-19 05:28:53,388 ] 94 main_controller - INFO - Subprocess 1 started with trial number 2
[ 2024-04-19 05:28:55,411 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:55,411 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:55,411 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:55,411 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:55,411 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:55,411 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:55,411 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:55,411 ] 94 main_controller - INFO - Browser opened
[ 2024-04-19 05:28:57,623 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:57,623 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:57,623 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:57,623 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:57,623 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:57,623 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:57,623 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:57,623 ] 82 main_controller - ERROR - Caught exception: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 71, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 23, in find
    self._collect(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 35, in _collect
    form_table = browser.find_element(By.XPATH, "/html/body/form/main/div/div[5]/div/ul")
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "c:\ProgramData\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/form/main/div/div[5]/div/ul"}
  (Session info: chrome=123.0.6312.124); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF79D9D7032+63090]
	(No symbol) [0x00007FF79D942C82]
	(No symbol) [0x00007FF79D7DEC65]
	(No symbol) [0x00007FF79D82499D]
	(No symbol) [0x00007FF79D824ADC]
	(No symbol) [0x00007FF79D865B37]
	(No symbol) [0x00007FF79D84701F]
	(No symbol) [0x00007FF79D863412]
	(No symbol) [0x00007FF79D846D83]
	(No symbol) [0x00007FF79D8183A8]
	(No symbol) [0x00007FF79D819441]
	GetHandleVerifier [0x00007FF79DDD25AD+4238317]
	GetHandleVerifier [0x00007FF79DE0F70D+4488525]
	GetHandleVerifier [0x00007FF79DE079EF+4456495]
	GetHandleVerifier [0x00007FF79DAB0576+953270]
	(No symbol) [0x00007FF79D94E54F]
	(No symbol) [0x00007FF79D949224]
	(No symbol) [0x00007FF79D94935B]
	(No symbol) [0x00007FF79D939B94]
	BaseThreadInitThunk [0x00007FF94FE9257D+29]
	RtlUserThreadStart [0x00007FF95116AA48+40]

[ 2024-04-19 05:28:57,625 ] 94 main_controller - INFO - Subprocess 2 started with trial number 2
[ 2024-04-19 05:28:57,625 ] 94 main_controller - INFO - Subprocess 2 started with trial number 2
[ 2024-04-19 05:28:57,625 ] 94 main_controller - INFO - Subprocess 2 started with trial number 2
[ 2024-04-19 05:28:57,625 ] 94 main_controller - INFO - Subprocess 2 started with trial number 2
[ 2024-04-19 05:28:57,625 ] 94 main_controller - INFO - Subprocess 2 started with trial number 2
[ 2024-04-19 05:28:57,625 ] 94 main_controller - INFO - Subprocess 2 started with trial number 2
[ 2024-04-19 05:28:57,625 ] 94 main_controller - INFO - Subprocess 2 started with trial number 2
[ 2024-04-19 05:28:57,625 ] 94 main_controller - INFO - Subprocess 2 started with trial number 2
[ 2024-04-19 05:28:57,625 ] 94 main_controller - INFO - Subprocess 2 started with trial number 2
